### PR TITLE
Setup test scripts

### PR DESCRIPTION
**Pull request by dkryvi**
Install jest, enzyme dependencies.
Write tests for `<Button/>` and `<ActionBar/>` components.
Add test scripts.

### CODE REVIEW

File: src/components/ActionBar/ActionBar.jsx [hunk 1]
```diff
@@ -23,6 +23,9 @@ class ActionsBar extends PureComponent {
 
   static defaultProps = {
     isVisible: false,
+    isMicrophoneEnabled: false,
```
Comment (BesMarcell) [inline src/components/ActionBar/ActionBar.jsx:26]: this default props are predefined in reducer

Comment (dkryvi) [reply src/components/ActionBar/ActionBar.jsx:26]: I think we should separate our components and store. What do you think about it?

File: src/components/ActionBar/ActionBar.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+};
+
+describe('<ActionBar /> component', () => {
+  it('should render without crashing', () => {
+    const component = shallow(<ActionBar {...defaultRequiredProps} />);
+
+    expect(component).toHaveLength(1);
+  });
+
+  it('snould render <Mic /> component when microphone turned on', () => {
```
Comment (ok2ju) [inline src/components/ActionBar/ActionBar.spec.jsx]: typo `snould` instead of `should`

Comment (ok2ju) [inline src/components/ActionBar/ActionBar.spec.jsx]: typo `snould` instead of `should`

Comment (dkryvi) [reply src/components/ActionBar/ActionBar.spec.jsx]: [Done](https://github.com/interviewr/raven-client/pull/39/commits/8382cdc194cf77de251cb00c60e8a6abdfca45aa#diff-9e645d2f387bae8dd6a2d3cfad28d8e4R21)

**Review by ok2ju — APPROVED** [review]

**Pull request merged**